nipype/algorithms/modelgen.py

```diff
@@ -331,15 +331,19 @@ class SpecifyModel(BaseInterface):
      - amplitudes : lists of amplitudes for each event. This will be ignored by
        SPM's Level1Design.
 
-     The following two (tmod, pmod) will be ignored by any Level1Design class
-     other than SPM:
+     The following three (tmod, pmod, orth) will be ignored by any Level1Design class other than SPM:
 
      - tmod : lists of conditions that should be temporally modulated. Should
        default to None if not being used.
      - pmod : list of Bunch corresponding to conditions
        - name : name of parametric modulator
        - param : values of the modulator
        - poly : degree of modulation
+     - orth : lists of instructions to orthogonolise parametric regressors or 
+       not. Same as in SPM, use 'Yes' to indicate orthogonalisation, and 'No' 
+       to explicitly prevent it. Use None for conditions where it is not being 
+       used. Note that by default SPM will orthogonalise parametric regressors 
```

Comment:
These lines all have trailing whitespace. Could you clean that up?

Reply:
sure, will do that

Reply:
done in new commits